Doc/library/os.path.rst

```diff
@@ -488,6 +488,26 @@ the :mod:`glob` module.)
       Accepts a :term:`path-like object`.
 
 
+.. function:: splitroot(path)
+
+   Split the pathname *path* into a triad ``(drive, root, tail)`` where:
+
+   1. *drive* is an optional mount point, exactly like :func:`splitdrive`;
+   2. *root* is an optional sequence of separators following the drive; and
+   3. *tail* is anything after the root.
+
+   On Posix, *drive* is always empty. The *root* may be empty (relative path),
+   a single forward slash (absolute path), or two forward slashes
+   (implementation-defined per the POSIX standard).
```

Comment:
What's the "POSIX standard"? Can that be linked to?

Reply:
POSIX stands for Portable Operating System Interface ([POSIX.1 FAQ](https://www.opengroup.org/austin/papers/posix_faq.html)). The following is the relevant statement from the [pathname resolution](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_13) discussion in IEEE Std 1003.1-2017:

> If a pathname begins with two successive \<slash\> characters, the first component following the leading \<slash\> characters may be interpreted in an implementation-defined manner, although more than two leading \<slash\> characters shall be treated as a single \<slash\> character.